Sources/XCTest/XCTAssert.swift

```diff
@@ -267,7 +267,7 @@ public func XCTAssertNil(@autoclosure expression: () -> Any?, @autoclosure _ mes
         if value == nil {
             return .Success
         } else {
-            return .ExpectedFailure("\"\(value)\"")
+            return .ExpectedFailure("\"\(value!)\"")
```

Comment:
I just reverted it to the original ordering at @eschaton's request. AFAIK there is no clean way to avoid the force-unwrap when doing the `nil` check first like this. Using `guard` could allow keeping the ordering the same, but I feel it be a pretty bad abuse of the expected semantics of that keyword.

Is there an alternative I'm overlooking? @modocache